hawkular-wildfly-agent-feature-pack/src/main/resources/subsystem-templates/hawkular-wildfly-agent.xml

```diff
@@ -30,7 +30,7 @@
                  timeUnits="minutes"/>
 
     <storage-adapter type="HAWKULAR"
-                     tenantId="${hawkular.rest.tenantId}"
+                     tenantId="${hawkular.rest.tenantId:hawkular}"
```

Comment:
I wonder why this has the `.rest` part all over (as with user name and password below). Do we expect othere for the future? @jmazzitelli